app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -642,9 +653,8 @@ public void onBookmarksHidden() {
     @Override
     public void onBookmarksClicked() {
         if (mBookmarksView.getVisibility() == View.VISIBLE) {
-            SessionStore.get().unstackSession();
-
             unsetView(mBookmarksView);
+            SessionStore.get().unstackSession();
```

Comment:
I realize this is existing code from the feature, but why do we need to push an empty session onto the stack to show book marks? It seems that this session is never actually used.